src/sentry/integrations/github/client.py

```diff
@@ -78,48 +126,45 @@ def get_with_pagination(self, path, *args, **kwargs):
             output.extend(resp)
             page_number = 1
 
-            def get_next_link(resp):
-                link = resp.headers.get("link")
-                if link is None:
-                    return None
-
-                # Should be a comma separated string of links
-                links = link.split(",")
-
-                for link in links:
-                    # If there is a 'next' link return the URL between the angle brackets, or None
-                    if 'rel="next"' in link:
-                        return link[link.find("<") + 1 : link.find(">")]
-
-                return None
-
             while get_next_link(resp) and page_number < self.page_number_limit:
                 resp = self.get(get_next_link(resp))
                 output.extend(resp)
                 page_number += 1
             return output
 
-    def get_issues(self, repo):
-        return self.get(f"/repos/{repo}/issues")
+    def get_issues(self, repo: str) -> Sequence[JSONData]:
+        issues: Sequence[JSONData] = self.get(f"/repos/{repo}/issues")
+        return issues
 
-    def search_issues(self, query):
-        return self.get("/search/issues", params={"q": query})
+    def search_issues(self, query: str) -> Mapping[str, Sequence[Mapping[str, Any]]]:
+        # Explicitly typing to satisfy mypy.Users/marcos/sentry/sentry/src/sentry/integrations/bitbucket/issues
```

Comment:
?

Reply:
This looks like a bad copy/paste. I've been having really bad input delays on my machine recently :/